awx/main/signals.py

```diff
@@ -235,14 +235,6 @@ def save_related_job_templates(sender, instance, **kwargs):
 
 
 def connect_computed_field_signals():
-    post_save.connect(emit_update_inventory_on_created_or_deleted, sender=Host)
```

Comment:
Eh, we probably need these for inv-level counts to be accurate.

Reply:
We could also enforce this using some mechanism in the dispatcher _itself_ i.e., make it so that for _certain_ tasks, the dispatcher throws away messages if one with matching args is _already_ queued.

Reply:
Maybe we could use https://docs.djangoproject.com/en/2.1/ref/signals/#request-finished to instead set some _flag_ on `request.environ`, and a middleware that would look for it at the _bottom_ of the request which meant "go update this inventory"

Reply:
Of course, this doesn't fix it for scenarios where you run a _ton_ of separate API requests that insert a bunch of hosts into a separate inventory and we end up firing off one task per request.

Reply:
I don't know that we ever emitted multiple tasks inside of a single request. Building an inventory through the API is a bunch of create & attach commands, so those appear to be 1-to-1 request-to-task.